fiftyone/core/odm/dataset.py

```diff
@@ -198,13 +205,19 @@ class ColorScheme(EmbeddedDocument):
                 document
             -   `valueColors` (optional): a list of dicts specifying colors to
                 use for individual values of this field
+        label_tags (None): an optional dict specifying custom colors for label tags
+            with the following keys:
+            -    `fieldColor` (optional): a color to assign to all label tags
+            -    `valueColors` (optional): a list of dicts specifying colors to
```

Comment:
We should not have camel case data sitting in MongoDB. If this is already the case, we should add a migration in the next release

Reply:
Hmm, they already exist now. I want to merge this PR, as there are bug fixes that is touching the same files, and I will raise a separate PR to fix camel case issue and migration.

Reply:
This is already the case. I will raise a seperate PR for the case conversion and migration.